content/id/docs/contribute/participate/_index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Berpartisipasi dalam SIG Docs
+content_type: concept
+weight: 60
+card:
+  name: contribute
+  weight: 60
+---
+
+<!-- overview -->
+
+SIG Docs merupakan salah satu
+[kelompok peminatan khusus (_special interest groups_)](https://github.com/kubernetes/community/blob/master/sig-list.md)
+dalam proyek Kubernetes, yang berfokus pada penulisan, pembaruan, dan pemeliharaan
+dokumentasi untuk Kubernetes secara keseluruhan. Lihatlah
+[SIG Docs dari repository github komunitas](https://github.com/kubernetes/community/tree/master/sig-docs)
+untuk informasi lebih lanjut tentang SIG.
+
+SIG Docs menerima konten dan ulasan dari semua kontributor. Siapa pun dapat membuka
+_pull request_ (PR), dan siapa pun boleh mengajukan masalah (_issue_) tentang konten atau komentar
+pada _pull request_ yang sedang berjalan.
+
+Kamu juga bisa menjadi [anggota (_member_)](/id/docs/contribute/participating/roles-and-responsibilities/#anggota),
+[pengulas (_reviewer_](/id/docs/contribute/participating/roles-and-responsibilities/#pengulas), atau [pemberi persetujuan (_approver_)](/id/docs/contribute/participating/roles-and-responsibilities/#approvers). Peran tersebut membutuhkan
+akses dan mensyaratkan tanggung jawab tertentu untuk menyetujui dan melakukan perubahan.
+Lihatlah [keanggotaan-komunitas (_community-membership_)](https://github.com/kubernetes/community/blob/master/community-membership.md)
+untuk informasi lebih lanjut tentang cara kerja keanggotaan dalam komunitas Kubernetes.
+
+Selebihnya dari dokumen ini akan menguraikan beberapa cara unik dari fungsi peranan tersebut dalam
+SIG Docs, yang bertanggung jawab untuk memelihara salah satu aspek yang paling berhadapan dengan publik
+dalam Kubernetes - situs web dan dokumentasi dari Kubernetes.
+
+
+<!-- body -->
+
+## Ketua umum (_chairperson_) SIG Docs {#ketua-umum-sig-docs}
+
+Setiap SIG, termasuk SIG Docs, memilih satu atau lebih anggota SIG untuk bertindak sebagai
+ketua umum. Mereka merupakan kontak utama antara SIG Docs dan bagian lain dari
+organisasi Kubernetes. Mereka membutuhkan pengetahuan yang luas tentang struktur
+proyek Kubernetes secara keseluruhan dan bagaimana SIG Docs bekerja di dalamnya. Lihatlah
+[Kepemimpinan (_leadership_)](https://github.com/kubernetes/community/tree/master/sig-docs#leadership)
+untuk daftar ketua umum yang sekarang.
+
+## Tim dan automasi dalam SIG Docs
+
+Otomasi dalam SIG Docs bergantung pada dua mekanisme berbeda:
```

Comment:
nit: let's be consistent on choice of words
```suggestion
Automasi dalam SIG Docs bergantung pada dua mekanisme berbeda:
```

Reply:
I think I confuse myself :)